mlflow/sagemaker/__init__.py

```diff
@@ -2518,7 +2518,7 @@ def predict(self, deployment_name, df):
             )
             response = sage_client.invoke_endpoint(
                 EndpointName=deployment_name,
-                Body=json.dumps(_get_jsonable_obj(df)),
+                Body=json.dumps(_get_jsonable_obj(df, pandas_orient="split")),
                 ContentType="application/json",
```

Comment:
Note: The `application/json` content type should be fine for all Pandas DataFrame orientations because the scoring server uses the data type of the payload (dict or list) to determine whether to parse the Pandas DataFrame from records or split format. See https://livegrep.dev.databricks.com/view/mlflow/mlflow/mlflow/pyfunc/scoring_server/__init__.py?commit=01182994ec31966d#L93-105

Reply:
So we don't need add "format=pandas-records" in content type ?

Reply:
seems `CONTENT_TYPE_JSON_RECORDS_ORIENTED` only used in testing code...